core/src/models/Destination.ts

```diff
@@ -492,6 +493,10 @@ export class Destination extends LoggedModel<Destination> {
     return DestinationOps.sendExports(this, _exports, sync);
   }
 
```

Comment:
Any extra validations needed?

Reply:
For example, perhaps a Destination cannot be deleted unless all the export processors for this destination are not in the pending state?

Reply:
Updated the existing check to not delete the destination if there are exports pending to also consider exports that are processing. I think that covers it.